superset-frontend/src/explore/components/SaveModal.tsx

```diff
@@ -183,6 +182,14 @@ class SaveModal extends React.Component<SaveModalProps, SaveModalState> {
       }
     }
 
+    //  Get chart dashboards
+    let sliceDashboards: number[];
+    promise = promise
+      .then(() => this.props.actions.getSliceDashboards(this.props.slice))
+      .then(dashboards => {
+        sliceDashboards = dashboards;
+      });
+
```

Comment:
nits:
```suggestion
    let sliceDashboards = await this.props.actions.getSliceDashboards(
      this.props.slice,
    );
```

Reply:
Oops, I didn't realize this `saveOrOverwrite` was now an `async` function!  I can clean up a lot of these promises now, thanks.

Reply:
I'm actually going to leave out turning the promises into `await`s because my full PR that included that fix triggered an Explore refresh bug that I think we're going to work on separately.